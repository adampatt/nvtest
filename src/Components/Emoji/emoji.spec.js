import React from "react";
import renderer from "react-test-renderer";
import Emoji from "./index";

it("renders when there are no inputs", () => {
	const tree = renderer
		.create(<Emoji />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

it("renders when there is a label added", () => {
	const label = "tick";
	const tree = renderer
		.create(<Emoji label={label} />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

it("renders when there is a label and symbol input", () => {
	const label = "tick";
	const symbol = "âœ…";
	const tree = renderer
		.create(
			<Emoji label={label} symbol={symbol} />
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
