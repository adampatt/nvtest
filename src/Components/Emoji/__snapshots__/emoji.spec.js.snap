// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no inputs 1`] = `
<span
  aria-hidden="true"
  aria-label=""
  className="emoji"
  role="img"
/>
`;

exports[`renders when there is a label added 1`] = `
<span
  aria-hidden="false"
  aria-label="tick"
  className="emoji"
  role="img"
/>
`;

exports[`renders when there is a label and symbol input 1`] = `
<span
  aria-hidden="false"
  aria-label="tick"
  className="emoji"
  role="img"
>
  âœ…
</span>
`;
